### PR TITLE
May 2023 Release

### PR DESCRIPTION
**Pull request by hectormmg**
This PR:
- Parses changefiles into CHANGELOG files for libraries due for major version upgrades
- Appends `-alpha.0` to package versions to give new major upgrades pre-release version numbers
- Integrates the msal-common changelogs into `msal-browser`, `msal-node` changelogs where necessary

### CODE REVIEW

File: change/@azure-msal-node-extensions-39583c2f-b5ce-47e6-bd13-35e6b463fe78.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```
Comment (tnorling) [inline change/@azure-msal-node-extensions-39583c2f-b5ce-47e6-bd13-35e6b463fe78.json]:
```suggestion
  "type": "none",
```

File: extensions/msal-node-extensions/package.json [hunk 2]
```diff
@@ -68,5 +68,8 @@
   },
   "bundledDependencies": [
     "@azure/msal-common"
+  ],
+  "bundleDependencies": [
```
Comment (tnorling) [inline extensions/msal-node-extensions/package.json:69]: Looks like "bundledDependencies" is already here. We should stick to one or the other

**Review by peterzenz — APPROVED** [review]

File: lib/msal-angular/package.json [hunk 3]
```diff
@@ -49,7 +49,7 @@
     "@angular/platform-browser": "^15.1.4",
     "@angular/platform-browser-dynamic": "^15.1.4",
     "@angular/router": "^15.1.4",
-    "@azure/msal-browser": "^2.32.2",
+    "@azure/msal-browser": "^3.0.0",
```
Comment (tnorling) [inline lib/msal-angular/package.json]:
```suggestion
    "@azure/msal-browser": "^3.0.0-alpha.0",
```

File: lib/msal-angular/package.json [hunk 4]
```diff
@@ -69,7 +69,7 @@
     "zone.js": "~0.11.8"
   },
   "peerDependencies": {
-    "@azure/msal-browser": "^2.35.0",
+    "@azure/msal-browser": "^3.0.0",
```
Comment (tnorling) [inline lib/msal-angular/package.json]:
```suggestion
    "@azure/msal-browser": "^3.0.0-alpha.0",
```

File: lib/msal-browser/package.json [hunk 5]
```diff
@@ -95,5 +95,8 @@
   },
   "bundledDependencies": [
     "@azure/msal-common"
+  ],
+  "bundleDependencies": [
```
Comment (tnorling) [inline lib/msal-browser/package.json:96]: Same comment here regarding bundle vs bundled

File: lib/msal-node/CHANGELOG.md [hunk 6]
```diff
@@ -1,9 +1,45 @@
...
+## 2.0.0-alpha.0
+
+Tue, 02 May 2023 23:14:26 GMT
+
+### Major changes
+
+- Move node clients from msal-common to msal-node #5788 (kshabelko@microsoft.com)
+- Migrate from tsdx to use rollup and jest directly for msal-node #5818 (hemoral@microsoft.com)
+- Exception is thrown in acquireTokenByClientCredential if tenantId is missing #5805 (rginsburg@microsoft.com)
+- Bump @azure/msal-common to v14.0.0
```
Comment (tnorling) [inline lib/msal-node/CHANGELOG.md]:
```suggestion
```

File: lib/msal-react/package.json [hunk 7]
```diff
@@ -39,12 +39,12 @@
     "format:fix": "npx prettier --ignore-path .gitignore --write src test"
   },
   "peerDependencies": {
-    "@azure/msal-browser": "^2.35.0",
+    "@azure/msal-browser": "^3.0.0",
```
Comment (tnorling) [inline lib/msal-react/package.json]:
```suggestion
    "@azure/msal-browser": "^3.0.0-alpha.0",
```

File: lib/msal-react/package.json [hunk 8]
```diff
@@ -39,12 +39,12 @@
...
     "react": "^16.8.0 || ^17 || ^18"
   },
   "module": "dist/msal-react.esm.js",
   "devDependencies": {
-    "@azure/msal-browser": "^2.35.0",
+    "@azure/msal-browser": "^3.0.0",
```
Comment (tnorling) [inline lib/msal-react/package.json]:
```suggestion
    "@azure/msal-browser": "^3.0.0-alpha.0",
```

File: lib/msal-browser/CHANGELOG.md [hunk 9]
```diff
@@ -1,9 +1,49 @@
...
 
+## 3.0.0-alpha.0
+
+Tue, 02 May 2023 23:14:26 GMT
+
+### Major changes
+
+- `allowNativeBroker` configuration defaults to `true` #5616 (hemoral@microsoft.com)
+- Fix telemetry typos #5868 (kshabelko@microsoft.com)
+- Bump @azure/msal-common to v14.0.0
```
Comment (tnorling) [inline lib/msal-browser/CHANGELOG.md]:
```suggestion
```

**Review by peterzenz — APPROVED** [review]

**Review by tnorling — APPROVED** [review]

**Review by konstantin-msft — APPROVED** [review]

**Pull request merged**